src/toast/ops/scan_healpix.py

```diff
@@ -242,7 +242,8 @@ class ScanHealpixMask(Operator):
     )
 
     det_flags_value = Int(
-        1, help="The detector flag value to set where the mask result is non-zero"
+        defaults.det_mask_processing,
+        help="The detector flag value to set where the mask result is non-zero",
```

Comment:
We should add this change to the `ScanWCSMask` operator as well.

Reply:
Done